marklogic-client-api/src/main/java/com/marklogic/client/dataservices/CallBatcher.java

```diff
@@ -279,7 +279,16 @@
          * @return  a CallBatcher that accepts the input and makes calls with the caller
          */
         CallBatcher<CallManager.CallArgs,E> forArgs();
-        
+        /**
+         * Takes a callback for generating the arguments for calls. 
+         * The callback takes the CallArgs from the previous call in the thread and returns 
```

Comment:
Is the input to the callback just the call args or the call event returned from the successful call?